From 517ddefc600ed06027e44d186788bbcba7c26b92 Mon Sep 17 00:00:00 2001
From: Andreas Cord-Landwehr <cordlandwehr@kde.org>
Date: Tue, 5 Mar 2024 19:02:16 +0100
Subject: [PATCH] Fix compilation with GCC 9

Upstream-Status: Backport [https://invent.kde.org/frameworks/ki18n/-/merge_requests/111]

Signed-off-by: Andreas Cord-Landwehr <cordlandwehr@kde.org>
---
 src/localedata/mapentry_p.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/localedata/mapentry_p.h b/src/localedata/mapentry_p.h
index 52aac90..f81d4ba 100644
--- a/src/localedata/mapentry_p.h
+++ b/src/localedata/mapentry_p.h
@@ -41,7 +41,7 @@ constexpr inline bool operator<(KeyType lhs, MapEntry<KeyType> rhs)
 template<typename MapEntry, std::size_t N>
 inline constexpr bool isSortedLookupTable(const MapEntry (&map)[N])
 {
-#if __cplusplus > 201703L && defined(__GNUC__) && !defined(__clang__)
+#if __cplusplus > 201703L && defined(__GNUC__) && __GNUC__ > 9 && !defined(__clang__)
     return std::is_sorted(std::begin(map), std::end(map));
 #else
     (void)map;
-- 
2.40.1

